pc2_bits.yml

```diff
@@ -27,16 +27,8 @@
         key: "{{ lookup('file', 'files/pc2.authorized_keys2') }}"
 
 
-- name: Configure wf46 pc2 machines
-  hosts: pc2server-wf46,ccsadmin-wf46,pc2-aj-wf46
-
-  roles:
-    - role: pc2client
-
-- name: Configure wf47 pc2 machines
-  hosts: pc2server-wf47,ccsadmin-wf47,pc2-aj-wf47
-  vars:
-    pc2server: 10.3.3.221
+- name: Configure pc2 machines
```

Comment:
What is s this for?  the server is in the cloud and does not have to be configured.  I think this was let over from 47 maybe?

Reply:
for Astana